addons/mail/static/tests/discuss/call/ptt_ad_banner.test.js

```diff
@@ -29,5 +31,25 @@ test("display banner when ptt extension is not enabled", async () => {
     await click("[title='Start a Call']");
     await contains(".o-discuss-PttAdBanner");
     await click("[title='toggle push-to-talk']");
+});
+
+test("mobile: do not display banner", async () => {
+    mockGetMedia();
+    const pyEnv = await startServer();
+    pyEnv["discuss.channel"].create({ name: "General" });
+    mockService("discuss.ptt_extension", {
+        get isEnabled() {
+            return false;
+        },
+    });
+    patchUiSize({ size: SIZES.SM });
+    await start({ device: "mobile" });
+    await click(".o_menu_systray i[aria-label='Messages']");
+    await click(".o-mail-NotificationItem", { text: "General" });
+    await click("[title='Start a Call']");
+    await click("[title='Open Actions Menu']");
+    await click("[title='Show Call Settings']");
+    await click("[title='toggle push-to-talk']");
+    await click("[title='Hide Call Settings']");
     await contains(".o-discuss-PttAdBanner", { count: 0 });
```

Comment:
Again would drop this test. Value is far too low and testing negative is not great.

The rule of thumb is testing core features to ensure they work. Hiding some content just for very minor QoL like these are really not worth the maintenance cost of test. If they are mistakenly shown this barely have any impact on usability of the product, and fixing it is extremely trivial.